### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by rekhakilari**
- [DEMO LINK](https://rekhakilari.github.io/layout_product-cards-en/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_product-cards-en/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+}
```
Comment (AndreiSuc) [inline src/style.css:124]: Your page doesn't use `style.css`. You should probably remove it to avoid confusion.

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,58 @@
...
+        <p class="product__text">Price:</p>
+        <p class="product__number">$2,199</p>
+      </div>
+
+      <a
+        href="#"
+        class="product__buy"
+        data-qa="hover"
+      >
+        BUY
```
Comment (AndreiSuc) [inline src/index.html:53]: It is not recommended to hardcode uppercase. JFYI.

**Review by AndreiSuc — APPROVED** [review]
Very clean work. Just please remove the unused `style.css` file.

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,58 @@
...
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="review__num">
+        Reviews: 5
```
Comment (ihor-jpeg) [inline src/index.html:39]: indentation

File: src/index.html (hunk 2 above)
Comment (ihor-jpeg) [reply src/index.html:53]: agreed

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 1rem;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 16px 4px;
```
Comment (ihor-jpeg) [inline src/style.css:60]: instead of setting horizontal margin why not give the entire card a horizontal padding?

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  text-decoration: none;
+  background-color: #00acdc;
+  margin: 0 16px 16px;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  transition: 0.3s;
```
Comment (ihor-jpeg) [inline src/style.css:114]: specify which properties you set a transition for

File: src/styles/blocks/product.css [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  color: #060b35;
+}
+
+.product__code {
+  align-self: flex-start;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0;
```
Comment (ihor-jpeg) [inline src/styles/blocks/product.css:31]: reset css in one place

**Review by ihor-jpeg — CHANGES_REQUESTED** [review]
Good job! Take a look at the comments

**Pull request closed without merging**